src/Neo/Plugins/Plugin.cs

```diff
@@ -205,15 +207,14 @@ private static void LoadPlugin(Assembly assembly, string? sourcePath = null)
         internal static void LoadPlugins()
         {
             if (!Directory.Exists(PluginsDirectory)) return;
-            List<(Assembly Assembly, string FileName)> assemblies = [];
+            List<Assembly> assemblies = [];
             foreach (var rootPath in Directory.GetDirectories(PluginsDirectory))
             {
                 foreach (var filename in Directory.EnumerateFiles(rootPath, "*.dll", SearchOption.TopDirectoryOnly))
                 {
                     try
                     {
-                        var assembly = Assembly.Load(File.ReadAllBytes(filename));
-                        assemblies.Add((assembly, filename));
+                        assemblies.Add(Assembly.Load(File.ReadAllBytes(filename)));
```

Comment:
```suggestion
                        assemblies.Add(Assembly.LoadFile(filename));
```
otherwise assembly doesn't have the path

Reply:
https://github.com/neo-project/neo-node/blob/d91898c631e0eab02b6a9caa7a9e71a3fd71fe89/src/Neo.CLI/CLI/MainService.Plugins.cs#L182